src/components/TodoItem.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import classNames from 'classnames';
+import React from 'react';
+
+type TodoItemProps = {
+  id: number;
+  completed: boolean;
+  title: string;
+  updatedTodosId: number[];
+  removeOneTodo: (id: number) => void;
+};
+
+export const TodoItem = ({
+  id,
+  completed,
+  title,
+  updatedTodosId,
+  removeOneTodo,
```

Comment:
There is a convention of passing a handler through props by calling it `onRemove`.